translations/harbour-todolist-zh_CN.ts

```diff
@@ -0,0 +1,637 @@
+        <translation>该日忽略</translation>
+    </message>
+    <message>
+        <source>to be done next day</source>
+        <translation>下一日要完成</translation>
+    </message>
+    <message>
+        <source>to be done someday later</source>
+        <translation>稍后某日要完成</translation>
+    </message>
+    <message>
+        <source>done this day</source>
+        <translation>该日完成</translation>
+    </message>
+    <message>
+        <source>handle today</source>
+        <translation>今日处理</translation>
+    </message>
+    <message>
+        <source>continue later this week</source>
+        <translation>本周稍后继续</translation>
+    </message>
+    <message>
+        <source>handle later this week</source>
+        <translation>本周稍后处理</translation>
+    </message>
+    <message>
+        <source>to be done later this week</source>
+        <translation>本周稍后要完成</translation>
+    </message>
+    <message>
+        <source>continue someday later</source>
+        <translation>稍后某日继续</translation>
+    </message>
+</context>
+<context>
+    <name>TodoListView</name>
+    <message>
+        <source>Todo List</source>
+        <translation>待办列表</translation>
+    </message>
+    <message>
+        <source>Show old entries</source>
+        <translation>显示旧事项</translation>
+    </message>
+    <message>
+        <source>About</source>
+        <translation>关于</translation>
+    </message>
+    <message>
+        <source>Note</source>
+        <translation>笔记</translation>
+    </message>
+    <message>
+        <source>Swipe left to add recurring entries. You can specify an interval in which they will be added automatically to the current todo list.</source>
+        <translation>左滑以添加重复事项。你可以指定自动添加到当前待办列表中的时间间隔。</translation>
+    </message>
+    <message>
+        <source>Add entry</source>
+        <translation>添加事项</translation>
+    </message>
+</context>
+<context>
+    <name>about</name>
+    <message>
+        <source>Programming</source>
+        <translation>编程</translation>
+    </message>
+    <message>
+        <source>English</source>
+        <translation>英语</translation>
+    </message>
+    <message>
+        <source>German</source>
+        <translation>德语</translation>
+    </message>
+    <message>
+        <source>Todo List</source>
+        <translation>待办列表</translation>
+    </message>
+    <message>
+        <source>A simple tool for planning what to do next.</source>
+        <translation>一个简单的工具，用于规划接下来要做什么。</translation>
+    </message>
+    <message>
+        <source>Sources on GitHub</source>
+        <translation>源代码位于 GiHub</translation>
+    </message>
+    <message>
+        <source>Icon Design</source>
+        <translation>图标设计</translation>
+    </message>
+</context>
+<context>
+    <name>harbour-todolist</name>
+    <message>
+        <source>d MMM yyyy &apos;(&apos;hh&apos;:&apos;mm&apos;)&apos;</source>
+        <translation>d MMM yyyy &apos;(&apos;hh&apos;:&apos;mm&apos;)&apos;</</translation>
```

Comment:
```suggestion
        <translation>d MMM yyyy &apos;(&apos;hh&apos;:&apos;mm&apos;)&apos;</translation>
```

I fixed and committed it; thanks for the translation!